.github/workflows/linux.yml

```diff
@@ -163,6 +163,8 @@ jobs:
                 -DCMAKE_CXX_COMPILER=${compilers[1]}       \
                 -G"Ninja"                                  \
                 -DYACLIB_LINK_OPTIONS="$link_option"       \
+                -DYACLIB_FAULT=OFF                         \
+                -DYACLIB_LOG="ERROR"                       \
```

Comment:
What do you think about disable/enable error check in release depends on some `rand`?

Reply:
Same for YACLIB_FAULT